R/data_remove.R

```diff
@@ -10,10 +10,10 @@ data_remove <- function(data, pattern, ignore_case = FALSE, verbose = FALSE, ...
   ## TODO set verbose = TRUE by default in a later update?
 
   # evaluate arguments
-  pattern <- .select_nse(pattern, data, exclude = NULL, ignore_case)
```

Comment:
@mattansb when the variable is named `pattern` instead of `select`, evaluation doesn't work... Any ideas?